browser-test/src/question_lifecycle.test.ts

```diff
@@ -11,30 +11,41 @@ describe('normal question lifecycle', () => {
 
     const questions = await adminQuestions.addAllNonSingleBlockQuestionTypes('qlc-');
     const singleBlockQuestions = await adminQuestions.addAllSingleBlockQuestionTypes('qlc-');
+    const repeatedQuestion = 'qlc-repeated-number';

```

Comment:
suggestion: Mind mentioning fixing this test in the PR description for future reference? I was confused why this was in here

Reply:
👍